source/sass/components/_form-custom-select.scss

```diff
@@ -0,0 +1,69 @@
+$select-height: 37px;
+
+.form-custom-select {
+  position: relative;
+  width: 100%;
+  min-width: 225px;
+  background-color: $color--white;
+  border: 1px solid $color--border;
+
+  @include media-query(tablet-portrait) {
+    max-width: 400px;
+  }
+
+  &:active,
+  &:focus {
+    border-color: $color--primary;
+  }
+
+  &::after {
+    content: "";
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    z-index: 2;
+    width: 50px;
+    height: 7px;
+    margin: auto 0;
+    pointer-events: none;
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24' viewBox='0 0 24 24'%3E%3Cpath d='M7.41 8.59L12 13.17l4.59-4.58L18 10l-6 6-6-6 1.41-1.41z'/%3E%3Cpath fill='none' d='M0 0h24v24H0V0z'/%3E%3C/svg%3E");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 24px 24px;
+  }
+
+  &::before {
+    content: "";
+    position: absolute;
+    top: 0;
+    right: 0;
+    z-index: 1;
+    display: block;
+    width: $select-height;
+    height: $select-height;
+    pointer-events: none;
+    background-color: #fff;
+  }
+
+  select {
+    @include font-size(xs);
+    -webkit-appearance: none;
```

Comment:
do you have autoprefixer? I thought this would have been generated by using `appearance: none;` which would give you the `-moz-` prefix as well as `-webkit-`. Either way, you'll need the `-moz-` one in there :)

Reply:
Thanks, this was me bug bashing, and forgot to remove (the actual select is hidden by the custom one, so the appearence doens't really matter :))! We do have autofixer, i'll get it removed